common/djangoapps/student/management/commands/manage_user.py

```diff
@@ -148,5 +148,10 @@ def handle(self, username, email, is_remove, is_staff, is_superuser, groups,
             )
         )
 
-        user.groups = new_groups
         user.save()
+
+        for grp in add_groups:
+            grp.user_set.add(user)
+
+        for grp in remove_groups:
+            grp.user_set.remove(user)
```

Comment:
Did you try `user.groups.set(new_groups)` as suggested in the docs for the deprecation? https://docs.djangoproject.com/en/2.2/releases/1.10/#direct-assignment-to-a-reverse-foreign-key-or-many-to-many-relation